node/errors.js

```diff
@@ -445,34 +444,71 @@ module.exports.classify = function classify(err) {
         case 'tchannel.max-pending-for-service':
             return 'Declined';
 
-        case 'tchannel.timeout':
+        case 'tchannel.socket-local-closed':
+        case 'tchannel.local.reset':
+            return 'Cancelled';
```

Comment:
Same feedback essentially as Kris: this isn't really a cancellation user action, in production I'd expect this to happen most frequently during upgrades / restarts / etc.